File: cpp/ZetCode/Qt4/01-Intro/simple.cpp
#include <QtGui/QApplication>
#include <QtGui/QWidget>


int main(int argc, char *argv[]) {
    QApplication app(argc, argv);

    QWidget window;

    window.resize(250, 150);
    window.setWindowTitle("Simple Example");
    window.show();

    return app.exec();
}

File: python/Zetcode/Advanced PyQt4 (PySide) Tutorial/04 - Graphics View Framework/simple.py
#!/usr/bin/env python

import sys
from PySide import QtGui, QtCore

class Example(QtGui.QGraphicsView):

    def __init__(self):
        QtGui.QGraphicsView.__init__(self)

        self.setGeometry(300, 300, 250, 150)
        self.setWindowTitle("Simple")
        self.init()

    def init(self):
        self.scene = QtGui.QGraphicsScene()
        self.scene.addText("ZetCode")

        self.setScene(self.scene)


app = QtGui.QApplication([])
root = Example()
root.show()
sys.exit(app.exec_())